index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<link rel="stylesheet" href="styles.css">
+<div>
+  <header id="header">
+    <div class="logo">
+    <img id="header-img" src=""><svg  viewBox="0 0 342 35" xmlns="http://www.w3.org/2000/svg"><path d="M0 .1a9.7 9.7 0 0 0 7 7h11l.5.1v27.6h6.8V7.3L26 7h11a9.8 9.8 0 0 0 7-7H0zm238.6 0h-6.8v34.8H263a9.7 9.7 0 0 0 6-6.8h-30.3V0zm-52.3 6.8c3.6-1 6.6-3.8 7.4-6.9l-38.1.1v20.6h31.1v7.2h-24.4a13.6 13.6 0 0 0-8.7 7h39.9v-21h-31.2v-7h24zm116.2 28h6.7v-14h24.6v14h6.7v-21h-38zM85.3 7h26a9.6 9.6 0 0 0 7.1-7H78.3a9.6 9.6 0 0 0 7 7zm0 13.8h26a9.6 9.6 0 0 0 7.1-7H78.3a9.6 9.6 0 0 0 7 7zm0 14.1h26a9.6 9.6 0 0 0 7.1-7H78.3a9.6 9.6 0 0 0 7 7zM308.5 7h26a9.6 9.6 0 0 0 7-7h-40a9.6 9.6 0 0 0 7 7z" fill="currentColor"></path></svg></img>
+    </div>
+
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#model3">Model3</a></li>
+        <li><a class="nav-link" href="#models"></a>Model S</li>
+        <li><a class="nav-link" href="#modely"></a>Model Y</li>
+        <li><a class="nav-link" href="#modelx"></a>Model X</li>
+        </ul>
+    </nav>
+    </header>
+
+    <form id="form" action="https://www.freecodecamp.com/email-submit" >
+    <p>Enter your email adress for a chance to win a Tesla:
+       <input id="email" type="email" placeholder="Email" name="email"></input>
+       <input id="submit" type="submit"/>
+       </form>
+
+    <section id="model3">
+      <iframe id="video" src="https://www.youtube.com/embed/Q4VGQPk2Dl8" title="Tesla Unveils Model 3" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+
+    <hr>
+
+    <section id="models">
+      <iframe id=video src="https://www.youtube.com/embed/9AhqDiARQMQ" title="Essentials | Model S" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+
+    <hr>
+
+    <section id="modely">
+       <iframe id="video" src="https://www.youtube.com/embed/Tb_Wn6K0uVs" title="Model Y Unveil" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+
+    <hr>
+
+    <section id="modelx">
+       <iframe id="video" src="https://www.youtube.com/embed/JFOOo3hEjw8" title="Essentials | Model X" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
```

Comment:
- [x] Kindly use the different class names in each section. I suggest being more creative with one another so it would be more fun while reviewing your code. 😄

Reply:
- [ ] Or while selecting the element in CSS you wanted to say :smile:

Reply:
That exactly wants I wanted to say 😆